src/auth/jwt-interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface JwtPayload {
+  userId: string;
+  email: string;
+}
```

Comment:
PrimathonAICodeReviewer

The `JwtPayload` interface looks well-defined for storing user ID and email in JWT payloads.
```diff
+
```


<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->